core/src/components/picker-legacy-column/test/standalone/picker-column.e2e.ts

```diff
@@ -4,13 +4,13 @@ import { configs, test } from '@utils/test/playwright';
 configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
   test.describe(title('picker-column'), () => {
```

Comment:
Do these titles need to be unique? If we change the internal ones but not these, will that cause any problems?

Reply:
Titles need to be unique within a file, so since the legacy and modern tests are separate files we should be 
fine.